### DISCUSSION TITLE
How to add the input to a manual trigger of an Activity Function

### DISCUSSION THREAD

**Discussion by LarsHanegraaf** [Azure/azure-functions-durable-extension#1847] [Q&A]
Hi all,

As described in #951 it is now possible to trigger an Activity Function manually using an [HTTP POST request](https://docs.microsoft.com/en-us/azure/azure-functions/functions-manually-run-non-http).

I followed the instructions of the Azure Durable Functions manual and I received a 202 response if do a POST request to my local Functions runtime. However in my terminal I see the following error that the activity function needs input that is `a DurableActivityContext or a String that represents the serialized input`.

I already tried adding the input as an argument in the url of the request and as the body of the JSON as key:value, but that didn't  resolve the errors.

Any ideas?

Thanks!
Lars

**Comment by ConnorMcMahon** [comment]
If your activity input type is a string, then the body should just be a string. Make sure the string is wrapped in quotes.

If your activity input type is a POCO, then the body should just be a JSON representation of that POCO object.
